app/controllers/api/groups_controller.rb

```diff
@@ -56,4 +56,8 @@ def accessible_records
     Queries::ExploreGroups.new
   end
 
+  def resource_options(scope:, serializer:, root:)
```

Comment:
cool. maybe leave a comment to highlight if/when someone is expecting same behaviour?